File: data_collector/src/aggregator/AggregateIntensivBetten.py
# handler.py

import numpy as np

import class.aggregationClient as aggregationClient
import helpers.s3Store as storeIntensivBettenClient


def aggregateIntensivBetten(event, context):
    client = aggregationClient()
    aggregation = client.aggregateIntensivBetten()

    s3Store = storeIntensivBettenClient("BUCKET_NAME")
    s3Store.storeCollectionToS3(betten)


if __name__ == "__main__":
    aggregateIntensivBetten('', '')

File: data_collector/src/aggregator/AggregateIntensivBetten.ts
import { logger } from "./helpers";
import { StoreIntensivBettenClient } from "./helpers/s3Store";
import { Lambda } from "aws-sdk";
import { AggregatorIntensivBettenClient } from "./class/aggregatorClient";
import { Aggregation } from "./models";

export const aggregateIntensivBetten = async () => {
  const client = new AggregatorIntensivBettenClient();

  const aggregate: any = await client.aggregateIntensivBetten();

  const s3Store = new StoreIntensivBettenClient(process.env.BUCKET_NAME_APP);
  await s3Store.storeCollectionToS3(aggregate);
  logger.info("Job completed successfully aggregated: ");
};
